### DISCUSSION TITLE
Esp backpack telemetry out

### DISCUSSION THREAD

**Discussion by Nordle420** [ExpressLRS/ExpressLRS#1343] [Ideas]
Hello,
Could the esp backpack send telemetry data over bluetooth? Like the s.port mirror some opentx radios can put out. 

Would be awesome feature imo. I always liked the telemetryviewer app for finding my lost gps models. And would be an easy way to interface with an antenna tracker.

**Comment by freeifier** [comment]
I was looking into the same thing. From which I know there might also be all hardware available on some TX modules to do that.
